theme/less/homepage.less

```diff
@@ -20,6 +20,26 @@ main article
   column-gap: 20px;
```

Comment:
header h1:
- Is smaller than in drupal
- Missing line-height to adjust subheader

Reply:
Done.